src/org/opendatakit/briefcase/export/FormDefinition.java

```diff
@@ -173,6 +197,17 @@ public static FormDefinition from(BriefcaseFormDefinition briefcaseFormDefinitio
         .collect(toMap(FormDefinition::controlFqn, e -> e));
   }
 
+  private static List<TreeElement> getAllDescendants(TreeElement e) {
```

Comment:
This could use `Stream.flatMap` like `XmlElement.flatten`. Will refactor if the overall solution is acceptable